packages/devtools_app/lib/src/primitives/utils.dart

```diff
@@ -1143,7 +1147,7 @@ Uri getServiceUriFromQueryString(String location) {
 /// Helper function to return the name of a key. If widget has a key
 /// use the key's name to record the select e.g.,
 ///   ga.select(MemoryScreen.id, ga.keyName(MemoryScreen.gcButtonKey));
-String keyName(Key key) => (key as ValueKey<String>)?.value;
+String keyName(Key key) => (key as ValueKey<String>).value;
```

Comment:
this method can be removed - we don't use it anywhere

Reply:
done